sim-lib/src/lib.rs

```diff
@@ -637,6 +648,7 @@ impl Simulation {
             generators.push(ExecutorKit {
                 source_info: node_info.clone(),
                 network_generator: network_generator.clone(),
+                activity_name: "Random activity".to_string(),
```

Comment:
This doesn't really add much, I'd just have no name for randomly generated activity.